pkg/server/config.go

```diff
@@ -84,6 +84,12 @@ func (s Server) NewCEDARClientCheck() {
 	s.checkRequiredConfig(appconfig.CEDARAPIKey)
 }
 
+// NewLambdaEndpointCheck checks if a lambda endpoint is configured
+func (s Server) NewLambdaEndpointCheck() string {
+	s.checkRequiredConfig(appconfig.LambdaEndpoint)
```

Comment:
Since it's a required config, I think we'll need to add something for `LAMBDA_ENDPOINT` to the `environment` section of the CircleCI `test` job: https://github.com/CMSgov/easi-app/blob/99f7239b973d7e3487d12f27e49fd4354d82abae/.circleci/config.yml#L90.

Reply:
@jsclarridge Do you think it should be required? I'm now thinking maybe not if we only need the value in one environment?

Reply:
If the endpoint isn't needed for the deployed environment or CI tests, then I think it's okay to leave it optional (i.e., not required).